app/src/main/res/layout/profile_edit_activity.xml

```diff
@@ -210,6 +210,7 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:fontFamily="sans-serif-medium"
+            android:layout_marginEnd="150dp"
```

Comment:
This is incorrect. Please do not introduce random margin/padding in UI.

To fix for this use following thing:
In line `197` update width to `android:layout_width="0dp"`
and in line `210` update width to `android:layout_width="match_parent"`

Reply:
Thanks for the help!

Reply:
@khushal1707 Please do not resolve comments. The person who has started the comment thread should resolve it.